contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -60,12 +60,24 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
     uint256 public currentReserveImbalance;
     uint256 public totalMinted;
 
+    // tokenMultiplier allows this contract to scale the number of tokens it sends/receives to/from
+    // the source chain. This can be used to normalize the number of decimals places between
+    // the two subnets.
+    uint256 public immutable tokenMultiplier;
+    // If multiplyOnSend is true, the number of tokens sent to the destination chain will be multiplied
+    // by `tokenMultiplier`, and tokens received from the source chain will be divided by `tokenMultiplier`.
+    // If multiplyOnSend is false, the number of tokens sent to the destination chain will be divided
+    // by `tokenMultiplier`, and tokens received from the source chain will be divided by `tokenMultiplier`.
+    bool public immutable multiplyOnSend;
```

Comment:
```suggestion
    // If multiplyOnReceive is true, the raw token amount value will be multiplied by `tokenMultiplier` when tokens
    // are transferred from the source chain into this destination chain, and divided by `tokenMultiplier` when
    // tokens are when tokens are transferred from this destination chain back to the source chain. This is intended 
    // when the "decimals" value on the source chain is less than the native EVM denomination of 18.
    // If multiplyOnReceive is false, the raw token amount value will be divided by `tokenMultiplier` when tokens
    // are transferred from the source chain into this destination chain, and multiplied by `tokenMultiplier` when
    // tokens are when tokens are transferred from this destination chain back to the source chain. This is intended 
    // when the "decimals" value on the source chain is greater than the native EVM denomination of 18.
    bool public immutable multiplyOnReceive;
```

I think the original comment mixed up "sent to destination" and "received from source", which are actually the same transfer direction.

Multiply on send vs receive is definitely a nit...that's just how I initially thought about it for the case where the source decimals is less than 18, which is likely more far common that when its greater than 18.